### PR TITLE
Release/4.0.0

### PR DESCRIPTION
**Pull request by bhupeshappfoster**
### Description:
Release v4.0.0 stable version for craft 5

### Related Issue(s):

- [Ad system](https://github.com/AcclaroInc/pm-craft-translations/issues/551)
- [Service page](https://github.com/AcclaroInc/pm-craft-translations/issues/552)
- [User registration](https://github.com/AcclaroInc/pm-craft-translations/issues/550)

### Changes Made:
[List the changes made in this pull request to address the issue or implement the feature.]

**Added:**

- 

**Changed:**

-

**Deprecated:**

-

**Removed:**

-

**Fixed:**

-

**Security:**

-

### Testing:
[Describe the testing performed to ensure the changes are functioning as expected.]

- 

### Quality Assurance (QA):

- [ ] The code has been reviewed and approved by the QA team.
- [ ] The changes have been tested on different environments (e.g., staging, production).
- [ ] Integration tests have been performed to verify the interactions between components.
- [ ] Performance tests have been conducted to ensure the changes do not impact system performance.
- [ ] Any necessary database migrations or data transformations have been executed successfully.
- [ ] Accessibility requirements have been considered and tested (e.g., screen reader compatibility, keyboard navigation).

### Resources:

**Screenshots (if applicable):**
[Include any relevant screenshots to visually demonstrate the changes.]

### Wrapping up

**Checklist:**

- [ ] The code builds without any errors or warnings.
- [ ] The code follows the project's coding conventions and style guidelines.
- [ ] Unit tests have been added or updated to cover the changes made.
- [ ] The documentation has been updated to reflect the changes (if applicable).
- [ ] All new and existing tests pass successfully.
- [ ] The PR has been reviewed by at least one other team member.

**Additional Notes:**
[Include any additional notes, considerations, or context that may be relevant.]

### CODE REVIEW

**Comment by bhupeshappfoster** [comment]
@sidedwards 

As feedback we have
- Changed the base branch to `release/4.0.0` for stable version of plugin.
- Updated the change log. (Ready to release)
- Waiting for final review before we roll out this craft5 compatible plugin version.

**NOTE:** These changes are deployed to dev instance of review.

**Review by sidedwards — APPROVED** [review]

**Pull request merged**
